tests/unit/Restoration.spec.ts

```diff
@@ -193,7 +222,21 @@ describe('Restoration component - edit page', () => {
     // create a Local Vue and install router on it
     const localVue = createLocalVue()
     localVue.use(VueRouter)
-    const router = mockRouter.mock()
+    const router: any = mockRouter.mock()
+    const store: any = getVuexStore()
```

Comment:
You don't have to specify type if the functions return a type.

If a function doesn't return a type then you should do, for example: `const x = getX() as number` (or whatever type).